Week3/HOMEWORK/js-exercises/project-creditCardValidator.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+console.log('JS1-W3-PROJECT-Credit Card Validator');
+
+function validateCreditCardNumber(cardNumber) {
+  // Credit card number must be 16 digits.
+  if (cardNumber.length !== 16) {
+    return `Error: INVALID Card Number! Your credit card number must be 16 digits!`;
+  }
+
+  // All of the credit card digits must be numbers.
+  let numbers = [];
+  if (typeof cardNumber === 'number') {
+    numbers = ('' + cardNumber).split('');
+  } else if (typeof cardNumber === 'string') {
+    numbers = cardNumber.split('');
+  } else {
+    // else
+    return `Error: INVALID Card Number!`;
+  }
```

Comment:
How about using a `parseInt` or `Number` here instead of so many if-else blocks. It would be easier, don't you think? 🙂